backend/src/utils/emails.py

```diff
@@ -0,0 +1,78 @@
+import smtplib
+from email.mime.text import MIMEText
+from email.header import Header
+
+
+def send_email(recipient_email, subject, body):
+    sender_email = "dssckwabot@gmail.com"
+    smtp_connection = smtplib.SMTP("smtp.gmail.com", 587)
+    smtp_connection.starttls()
+    smtp_connection.login(sender_email, "tgscgeeytaeegvvg")
```

Comment:
Ja bym nie trzymała tego w repo, można po prostu to wyeksportować do pliku lokalnego
No i trzeba zmienić to hasło teraz

Reply:
za dużo nas jest żebyśmy wymagali lokalnych plików 
to hasło najlepiej by było jakby było w secretach na githubie i było pobierane z envów ale szkoda czasu na konfigurowanie tego 
nie jest to mail nikogo z nas to bym to zostawił w ten sposób

Reply:
1. W poprzednim semestrze było nas tyle samo i bez problemu przeszedł lokalny plik
2. Repozytoria są cały czas skanowane przez boty, które m.in. szukają takich haseł -> raczej nie chcemy tworzyć nowego maila, bo zostanie zblokowany przez rozsyłanie jakiegoś spamu
3. Fajnie jakby było w secretach, ale szybciej zrobi się po prostu osobny plik, który nie będzie udostępniony do gita
4. Zamiast pliku można skonfigurować zmienną środowiskową
5. To repo jest publiczne, gdzie przechowywanie w taki sposób haseł na gicie nie jest zalecane nawet w przypadku repozytoriów prywatnych

Reply:
w poprzednim semestrze mniej osób kod odpalało niż teraz 
poza tym wg wymagań projektu kod musi się uruchamiać od razu po sklonowaniu a jak będzie wymagany dodatkowy plik/env to to nie przejdzie
szkoda marnować czasu na rozsyłanie między sobą pliku z hasłem jak są ważniejsze rzeczy do roboty

Reply:
W poprzednim semestrze było 5 osób w zespole i wszystkie odpalały kod
Jak zamierzasz uruchomić kod od razu po sklonowaniu, skoro trzeba importować bazę (a wcześniej zainstalować PostgreSQL)?

Reply:
było 5 osób w zespole i 3 odpalały kod po implementacji maili 
baza siedzi elegancko w repo i import przechodzi od 0 konfiguracji bez dodawania jakiś rzeczy z zewnątrz
idąc tym tokiem myślenia to pythona noda i windowsa też trzeba mieć zainstalowanego żeby to się odpaliło...

Reply:
No tak, trzeba mieć zainstalowane te rzeczy, dlatego nie da się zrobić `git clone` i od razu mieć uruchomioną aplikację

Reply:
no 
ale przygotowanie enva czy pliku w określonym miejscu żeby aplikacja działa jest czym innym niż posiadanie zainstalowanego pythona...

Reply:
Do czego zmierzasz?

Reply:
patrz mój 1 komentarz

Reply:
żeby odpalić apkę trzeba i tak mieć:
1. pythona
2. node'a
3. postgresa

i jak już ściągniesz ją, to wciąż trzeba:
1. zaimportować bazę danych
2. stworzyć venv
3. aktywować venv
4. zainstalować zależności backendu
5. odpalić backend
6. zainstalować zależności frontendu
7. odpalić frontend

więc dorzucenie punktu "uzupełnić plik config.json" niewiele zmieni

Reply:
a skąd weźmiesz prawidłową zawartość pliku? wyczarujesz sobie to hasło?

Reply:
Hasło można podesłać w inny sposób, a nie podawać publicznie.